android/src/main/java/io/fullstack/firestack/remoteconfig/FirestackRemoteConfig.java

```diff
@@ -0,0 +1,226 @@
+package io.fullstack.firestack.remoteconfig;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.NativeModule;
+
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig;
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings;
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigValue;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import io.fullstack.firestack.Utils;
+
+public class FirestackRemoteConfig extends ReactContextBaseJavaModule {
+    private static final String TAG = "FirestackRemoteConfig";
+    private FirebaseRemoteConfig mFirebaseRemoteConfig;
+
+    public FirestackRemoteConfig(ReactApplicationContext reactContext) {
+        super(reactContext);
+        mFirebaseRemoteConfig = FirebaseRemoteConfig.getInstance();
+        Log.d(TAG, "New FirestackRemoteConfig instance");
+    }
+
+    /**
+     * @return
+     */
+    @Override
+    public String getName() {
+        return TAG;
+    }
+
+    @Override
+    public Map<String, Object> getConstants() {
+        final Map<String, Object> constants = new HashMap<>();
+        constants.put("VALUE_SOURCE_DEFAULT", FirebaseRemoteConfig.VALUE_SOURCE_DEFAULT);
+        constants.put("VALUE_SOURCE_REMOTE", FirebaseRemoteConfig.VALUE_SOURCE_REMOTE);
+        constants.put("VALUE_SOURCE_STATIC", FirebaseRemoteConfig.VALUE_SOURCE_STATIC);
+        return constants;
+    }
+
+    private FirebaseRemoteConfigValue getConfig(String key, String namespace) {
+        if (namespace != null && !namespace.isEmpty()) {
+            return mFirebaseRemoteConfig.getValue(key, namespace);
+        } else {
+            return mFirebaseRemoteConfig.getValue(key);
+        }
+    }
+
+    @ReactMethod
+    public void fetch(final Integer cacheExpirationSeconds, final Promise promise) {
+        try {
+            Task fetchTask = mFirebaseRemoteConfig.fetch(cacheExpirationSeconds.longValue());
+            fetchTask.addOnCompleteListener(new OnCompleteListener<Void>() {
+                @Override
+                public void onComplete(@NonNull Task<Void> task) {
```

Comment:
@achintkalra thanks Achint, This code will run in Android, will that make any difference?